include/odp/api/spec/traffic_mngr.h

```diff
@@ -1584,6 +1584,12 @@ typedef struct {
 	 * have the same single strict priority level and this level must be
 	 * in the range 0..max_priority. */
 	uint8_t priority;
+
```

Comment:
The subject line of the commit message of this patch could be changed to be more descriptive so that it mentions also the addition of the multi packet enqueue function: 
Subject line now: api: tm: add option to do an unordered enqueue to TM queue
This would be better: api: tm: add unordered enqueue and multi-packet enqueue